mock_hardware/src/generic_system.cpp

```diff
@@ -636,9 +752,59 @@ return_type GenericSystem::read(const rclcpp::Time & /*time*/, const rclcpp::Dur
     mirror_command_to_state(gpio_states_, gpio_commands_);
   }
 
+  // do loopback over all transmission interface an update
+  // actuator: state -> transmission
+  std::for_each(
+    actuator_interfaces_.begin(), actuator_interfaces_.end(),
+    [](auto & actuator_interface)
+    { actuator_interface.transmission_passthrough_ = actuator_interface.state_; });
```

Comment:
Ah OK, this is just a storage. Can't we use directly `actuator_interface.state_`?

Reply:
no see above comment